addon/webextension/selector/ui.js

```diff
@@ -231,7 +231,11 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
                <body>
                  <div class="preview-overlay">
                    <div class="fixed-container">
-                     <div class="preview-instructions"></div>
+                     <div class="preview-instructions">
+                       <span class="preview-instructions-step-one"></span>
+                       &nbsp;
+                       <span class="preview-instructions-step-two"></span>
```

Comment:
Hey @stasm (or @flodolo), I'm not sure if this is the right way to put two strings on the same visual line such that it renders correctly in RTL languages: `span` + `&nbsp;` + `span`, and setting the `textContent` of each `span` to a different localized string.

Here's how this looks in regular English:

<img width="560" alt="screen shot 2017-03-22 at 1 41 22 pm" src="https://cloud.githubusercontent.com/assets/96396/24220064/32087462-0f06-11e7-82d1-c6b4e65ff8f2.png">

If I set `dir="rtl"` in the `body` tag on the iframe, English RTL looks like this:

<img width="548" alt="screen shot 2017-03-22 at 1 42 10 pm" src="https://cloud.githubusercontent.com/assets/96396/24220087/40f7b3f2-0f06-11e7-809e-da98b5ab33a0.png">

Does that look correct?

Finally, the one annoyance about this approach is that, when the screen is narrow, the strings wrap a bit awkwardly:

<img width="523" alt="screen shot 2017-03-22 at 1 41 31 pm" src="https://cloud.githubusercontent.com/assets/96396/24220107/5382ed02-0f06-11e7-9aaf-d7a98cdcf320.png">

Any suggestions there? I could concatenate the strings and shove them into a single element, but wasn't sure that would work for RTL languages. @johngruen may have ideas for nicer wrapping in that situation, too.